src/app/components/MostRead/Canonical/Item/index.tsx

```diff
@@ -46,14 +50,22 @@ export const MostReadLink = ({
   size,
   eventTrackingData,
 }: PropsWithChildren<MostReadLinkProps>) => {
-  const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+  const { isLite } = useContext(RequestContext);
+
+  const clickTrackerHandler = isLite
+    ? {
+        [LITE_TRACKER_PARAM]: useConstructLiteSiteUrl(eventTrackingData),
+      }
+    : {
+        onClick: useClickTrackerHandler(eventTrackingData),
+      };
```

Comment:
Should this logic be moved into the `useClickTrackerHandler` code, meaning that the only changes we would need to make to the other components which have click tracking would be what you've done in this file on line 68?